dbt_project.yml

```
@@ -31,3 +29,7 @@ on-run-end:
 models:
   logging:
     +schema: meta
+    bigquery:
+      +enabled: '{{ target.type == "bigquery" }}'
+    default:
+      +enabled: '{{ target.type != "bigquery" }}'
```

I think you need to `as_bool` these:
```suggestion
      +enabled: '{{ target.type == "bigquery" | as_bool }}'
    default:
      +enabled: '{{ target.type != "bigquery" | as_bool }}'
```